cmd/internal/cli/actions.go

```diff
@@ -136,7 +136,15 @@ func handleLibrary(u string) (string, error) {
 		return "", fmt.Errorf("unable to check if %v exists: %v", imagePath, err)
 	} else if !exists {
 		sylog.Infof("Downloading library image")
-		libexec.PullLibraryImage(imagePath, u, "https://library.sylabs.io", false, authToken)
+		if err = library.DownloadImage(imagePath, u, PullLibraryURI, true, authToken); err != nil {
```

Comment:
Sorry, one last nitpick. Lets leave libraryURL hardcoded as `https://library.sylabs.io` since action commands don't currently support a custom library url. Another option is to change line 127 to use `PullLIbraryURI`, but I think the former is clearer to a reader